ARC specification.md

```diff
@@ -296,102 +302,156 @@ Notes:
+          └── {$package}
   ```
   
   where:
   - `{$branch}` is the name of the branch the validation was run on
-  - `{$commithash}` is the full hash of the commit the validation was run on. See 
-  - `{$target}` is the name of the target the validation was run against. This folder then MUST contain the files `validation_report.*` and `badge.svg` as described [above](#mechanism-for-quality-control-of-arcs).
+  - `{$commithash}` is the full hash of the commit the validation was run on. 
+  - `{$package}` is the name of the validation package the validation was run against. 
+    this folder then MUST contain the files `validation_report.*` and `badge.svg` as described in the [validation package specification](#validation-packages).
 
   example:
 
-  > This example shows the validation results of the `main` and `branch-1` branches of the ARC repository against the `target1` and `target2` targets for two commits per branch:
+  > This example shows the validation results of the `main` and `branch-1` branches of the ARC repository against the `package1` and `package2` validation packages for two commits per branch:
 
   ```
-  validation-branch-root
+  cqc-branch-root
   ├── branch-1
   │   ├── ca82a6dff817ec66f44342007202690a93763949
-  │   │   ├── target1
+  │   │   ├── package1
   │   │   │   ├── badge.svg
   │   │   │   └── validation_report.xml
-  │   │   └── target2
+  │   │   └── package2
   │   │       ├── badge.svg
   │   │       └── validation_report.xml
   │   └── 085bb3bcb608e1e8451d4b2432f8ecbe6306e7e7
-  │       ├── target1
+  │       ├── package1
   │       │   ├── badge.svg
   │       │   └── validation_report.xml
-  │       └── target2
+  │       └── package2
   │           ├── badge.svg
   │           └── validation_report.xml
   └── main
       ├── 1234567890abcdef1234567890abcdef12345678
-      │   ├── target1
+      │   ├── package1
       │   │   ├── badge.svg
       │   │   └── validation_report.xml
-      │   └── target2
+      │   └── package2
       │       ├── badge.svg
       │       └── validation_report.xml
       └── a11bef06a3f659402fe7563abf99ad00de2209e6
-          ├── target1
+          ├── package1
           │   ├── badge.svg
           │   └── validation_report.xml
-          └── target2
+          └── package2
               ├── badge.svg
               └── validation_report.xml
   ```
 
-### Structure of the validation_targets.yml file
+### The validation_packages.yml file
 
-The `validation_targets.yml` specifies the targets that the branch containing the file will be validated against. Each branch of an ARC MAY contain 0 or 1 `validation_targets.yml` file. If the file is present, it:
-  - MUST be located in an `.arc` folder in the root of the ARC
-  - MUST contain the `targets` key which is a list of target names that the current branch will be validated against.
+The `validation_packages.yml` specifies the validation packages that the branch containing the file will be validated against.
```

Comment:
validated against `in the cqc`.
?

Reply:
not necessarily. a local tool can take information from here as well